pages/index.js

```diff
@@ -15,10 +16,57 @@ export default function Home() {
   const language = items.language;
   const t = language === "en" ? en : fr;
 
-  const { data, error } = userSwr("/api/topics", fetcher);
+  const [topicsData, setTopicData] = useState(null);
+  const [errorCode, setErrorCode] = useState(false);
 
-  if (!data) return <div>Loading...</div>;
-  if (error === !null) return <div>Failed to load topics...</div>;
+  const { data, error } = useSwr("/api/topics", fetcher);
+
+  const ADOBE_ENABLED = process.env.ADOBE_ENABLED
+    ? process.env.ADOBE_ENABLED
+    : false;
+
+  //If on live instance, use AEM api for topic box content
+  if (ADOBE_ENABLED) {
+    useEffect(() => {
+      async function TopicList() {
+        const { errorCode, topicsData } = await getTopics("en");
+
+        let topics = [];
+
+        topicsData.entities.map((entity) => {
+          if (entity.properties.contentFragment) {
+            topics.push({
+              title: entity.properties.elements.title.value,
+              body: entity.properties.elements.description.value,
+              image: "/images/family.png",
+              imgalt: "Alt text TBD",
+              url: entity.properties.elements.link.value,
+              subtopics: entity.properties.elements.tags.value
+                .toString()
+                .split(","),
+            });
+          }
+        });
+
+        // console.log("topics = ", topics)
+        if (errorCode) setErrorCode(true);
+        setTopicData(topics);
+      }
+      TopicList();
+    }, []);
+
+    if (!topicsData)
+      return <div className="center text-h3">Loading Data ...</div>;
+
+    if (errorCode) return <div>Failed to load topics...</div>;
+  }
+  //
+  //If on dev instance, use data.js for topic box content
+  //
+  else {
+    if (!data) return <div>Loading...</div>;
+    if (error === !null) return <div>Failed to load topics...</div>;
+  }
```

Comment:
There's a console error in the browser dev tools around this. 

`Warning: Text content did not match. Server: "Loading Data ..." Client: "Loading..."`

I think we can squash the bug by making this say "Loading Data..."